### PR TITLE
Create 2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md

### LINKED ISSUE #366: Create blog post for December 2, 2019 about the `ember-cli-mirage` addon
## Goal
Write a short blog post for the December blog series that is similar to the [example post]( https://github.com/ember-learn/ember-blog/blob/be8f29ba0ae8b5367d1758b1a150c1b503812a77/source/2019-12-01-countdown-to-the-new-year-ember-sortable.md).

## Criteria

-  [x] file name should 2019-12-DD-countdown-to-the-new-year-ADDON-NAME.md where DD is replaced with the day and ADDON-NAME is replaced with the hyphenated name of the addon
- [x] The opening sentence should be updated where COUNT is the sequential ordinal word. 

```
<b>This is the COUNT in our DecEmber series- <i>"Countdown to The New Year: 31 Days of 
Ember Addons"</i>.  We plan to highlight a new addon each day until the new year, and 
we hope you'll join us for the fun!</b>
```

- [x] The first h2 should be "Day NUM" where NUM is the same as the DD in the date.
- [x] the link to the addon should point to the page on Ember Observer
- [x] There should be two sub sections: (both h3)- "What it does" and "Why I like it" 
- [ ] Include whether or not the addon has been updated yet for Ember Octane- if not, invite contributors to help get the addon Octane ready.
- [x] The closing sentence can be slightly altered but should feel similar: 
"Do you use this addon? Or one like it? We'd love to hear about Ember addons that bring you joy!"

### PR DESCRIPTION
**Pull request by MelSumner**
When merged, this will create a new blog post for the DecEmber 2019 blog series. 

Should be merged on December 2nd, 2019.

Resolves #366

### CODE REVIEW

File: source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year: Ember CLI Mirage
```
Comment (jayjayjpg) [inline source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md]: Do you think it makes to stick to the lower case style of addons in this series? E.g.

```suggestion
title: Countdown to The New Year: ember-cli-mirage
```

What are your thoughts?

**Review by jayjayjpg — COMMENTED** [review]
This reads great! I left one more suggestion - let me know what you think! :sparkles:

File: source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md (hunk 1 above)
Comment (MelSumner) [reply source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md]: not in the title of the article, but when referenced inside the article, yes.

File: source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year- Ember CLI Mirage
+author: Melanie Sumner
+tags: 2019, Addons, DecEmber
+alias: 2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md
+responsive: true
+---
+
+This is the second in our December series- "Countdown to The New Year: 31 Days of Ember Addons".  We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!
```
Comment (locks) [inline source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md]:
```suggestion
This is the second in our December series–"Countdown to The New Year: 31 Days of Ember Addons".  We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun! 
```

File: source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+tags: 2019, Addons, DecEmber
+alias: 2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md
+responsive: true
+---
+
+This is the second in our December series- "Countdown to The New Year: 31 Days of Ember Addons".  We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun! 
+
+## Day 2
+
+Today we are going to look at one of our community's most popular addons- [ember-cli-mirage](https://emberobserver.com/addons/ember-cli-mirage).
```
Comment (locks) [inline source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md:13]: Just curious, what's the popularity metric?

File: source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+Today we are going to look at one of our community's most popular addons- [ember-cli-mirage](https://emberobserver.com/addons/ember-cli-mirage). 
+
+### What It Does
+
+The `ember-cli-mirage` addon is a client-side server to help you build, test, and demo your Ember app. It helps developers be more productive by providing ways to generate fake data, allowing front-end engineers to spend more time focusing on the UI. 
+
+### Why I Like It
+
+As I was informally polling the community about their favorite Ember addons, ember-cli-mirage came up- A LOT. For good reason, too- it's well-supported by the addon maintainers, there's a dedicated [documentation site](https://www.ember-cli-mirage.com/), and there's support on the [Ember Community Discord server](https://discord.gg/emberjs) in the #ec-mirage channel.
```
Comment (locks) [inline source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md]:
```suggestion
As I was informally polling the community about their favorite Ember addons, ember-cli-mirage came up A LOT. For good reason, too. It's well-supported by the addon maintainers, there's a dedicated [documentation site](https://www.ember-cli-mirage.com/), and there's support on the [Ember Community Discord server](https://discord.gg/emberjs) in the #ec-mirage channel. 
```

File: source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+The `ember-cli-mirage` addon is a client-side server to help you build, test, and demo your Ember app. It helps developers be more productive by providing ways to generate fake data, allowing front-end engineers to spend more time focusing on the UI. 
+
+### Why I Like It
+
+As I was informally polling the community about their favorite Ember addons, ember-cli-mirage came up- A LOT. For good reason, too- it's well-supported by the addon maintainers, there's a dedicated [documentation site](https://www.ember-cli-mirage.com/), and there's support on the [Ember Community Discord server](https://discord.gg/emberjs) in the #ec-mirage channel. 
+
+> "It’s a part of the ecosystem I dearly miss when I work with other frameworks."
+
+As an extra bonus, EmberMap has generously made their ["Mirage Tips and Tricks" video series](https://embermap.com/topics/mirage-tips-and-tricks) free to everyone (usually for subscribers only!) for December- be sure to check it out!
```
Comment (locks) [inline source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md]:
```suggestion
As an extra bonus, EmberMap has generously made their ["Mirage Tips and Tricks" video series](https://embermap.com/topics/mirage-tips-and-tricks) free to everyone (usually for subscribers only!) for December. Be sure to check it out! 
```

Comment (locks) [inline source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md]:
```suggestion
As an extra bonus, EmberMap has generously made their ["Mirage Tips and Tricks" video series](https://embermap.com/topics/mirage-tips-and-tricks) free to everyone (usually for subscribers only!) for December–be sure to check it out! 
```

File: source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md (hunk 3 above)
Comment (MelSumner) [reply source/2019-12-02-countdown-to-the-new-year-ember-cli-mirage.md:13]: position in the Ember Observer top 100 addons chart. (it comes in at 15)

**Review by jayjayjpg — APPROVED** [review]
This reads great! Thank you for another :sparkles: December writeup this week!

**Comment by amyrlam** [comment]
Content lgtm! 

I added a change to the template and Dec 1st, if you want to include it: https://github.com/ember-learn/ember-blog/pull/396

**Pull request merged**